package.json

```diff
@@ -10,6 +10,7 @@
     "pretest": "npm run lint",
     "test": "karma start test/config/karma.js --single-run",
     "test:integration": ". ./.env && bundle exec rake sauce:spec test_files=spec",
+    "test:publishing": "npm run pretest && ./node_modules/mocha/bin/mocha test/publishing",
```

Comment:
You shouldn't need to do `./node_modules/mocha/bin/mocha`. `mocha` should use the node modules version when run from an npm script.